src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -32953,6 +32953,12 @@ public static void Main(int arg)
                 // (9,22): error CS0103: The name 'z2' does not exist in the current context
                 //         int x = z1 + z2;
                 Diagnostic(ErrorCode.ERR_NameNotInContext, "z2").WithArguments("z2").WithLocation(9, 22),
+                // (6,55): error CS0165: Use of unassigned local variable 'z1'
```

Comment:
>// (6,55): error CS0165: Use of unassigned local variable 'z1' [](start = 16, length = 62)

Not sure whether we want this diagnostics, given that the value should be constant.